pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -639,6 +640,111 @@ func TestNewFrameworkPluginDefaults(t *testing.T) {
+		want    map[framework.ClusterEvent]sets.String
+	}{
+		{
+			name:    "no-op plugin",
+			plugins: []framework.Plugin{&fakeNoopPlugin{}},
+			want: map[framework.ClusterEvent]sets.String{
+				{Resource: framework.Pod, ActionType: framework.All}:                   sets.NewString("fakeNoop"),
+				{Resource: framework.Node, ActionType: framework.All}:                  sets.NewString("fakeNoop"),
+				{Resource: framework.CSINode, ActionType: framework.All}:               sets.NewString("fakeNoop"),
+				{Resource: framework.PersistentVolume, ActionType: framework.All}:      sets.NewString("fakeNoop"),
+				{Resource: framework.PersistentVolumeClaim, ActionType: framework.All}: sets.NewString("fakeNoop"),
+				{Resource: framework.Service, ActionType: framework.All}:               sets.NewString("fakeNoop"),
+				{Resource: framework.StorageClass, ActionType: framework.All}:          sets.NewString("fakeNoop"),
+			},
+		},
+		{
+			name:    "node plugin",
+			plugins: []framework.Plugin{&fakeNodePlugin{}},
+			want: map[framework.ClusterEvent]sets.String{
+				{Resource: framework.Pod, ActionType: framework.All}:                           sets.NewString("fakeNode"),
+				{Resource: framework.Node, ActionType: framework.Delete}:                       sets.NewString("fakeNode"),
+				{Resource: framework.CSINode, ActionType: framework.Update | framework.Delete}: sets.NewString("fakeNode"),
+			},
+		},
+		{
+			name:    "pod plugin",
+			plugins: []framework.Plugin{&fakePodPlugin{}},
+			want: map[framework.ClusterEvent]sets.String{
+				{Resource: framework.Pod, ActionType: framework.All}:                     sets.NewString("fakePod"),
+				{Resource: framework.Node, ActionType: framework.Add | framework.Delete}: sets.NewString("fakePod"),
+				{Resource: framework.Service, ActionType: framework.Delete}:              sets.NewString("fakePod"),
+			},
+		},
+		{
+			name:    "node and pod plugin",
+			plugins: []framework.Plugin{&fakeNodePlugin{}, &fakePodPlugin{}},
+			want: map[framework.ClusterEvent]sets.String{
+				{Resource: framework.Node, ActionType: framework.Delete}:                       sets.NewString("fakeNode"),
+				{Resource: framework.Node, ActionType: framework.Add | framework.Delete}:       sets.NewString("fakePod"),
+				{Resource: framework.Pod, ActionType: framework.All}:                           sets.NewString("fakeNode", "fakePod"),
+				{Resource: framework.CSINode, ActionType: framework.Update | framework.Delete}: sets.NewString("fakeNode"),
+				{Resource: framework.Service, ActionType: framework.Delete}:                    sets.NewString("fakePod"),
+			},
+		},
+		{
+			name:    "no-op runtime plugin",
+			plugins: []framework.Plugin{&fakeNoopRuntimePlugin{}},
+			want:    map[framework.ClusterEvent]sets.String{},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := make(map[framework.ClusterEvent]sets.String)
+			for _, p := range tt.plugins {
+				fillEventToPluginMap(p, got)
```

Comment:
test NewFramework instead. It's fine that the test is focused in the effects of this function, but better to test the exported function.

Reply:
We can - it's just we have to cover queueSort and bind, so updated code.

Reply:
Test above looks good, we can remove this one.